From 27d7269b7179c82a836f6321b389c0c54f4d11a4 Mon Sep 17 00:00:00 2001
From: Christian Ehrhardt <christian.ehrhardt@codasip.com>
Date: Tue, 4 Mar 2025 09:36:00 +0100
Subject: qemu: Disable some tests that take too long on qemu

Disable several stress tests that take too long when run on qemu.

Upstream-Status: Inappropriate [ CHERI Specific ]

diff --git a/runtest/controllers b/runtest/controllers
index 93c52c439..d0ad2d7fa 100644
--- a/runtest/controllers
+++ b/runtest/controllers
@@ -39,306 +39,306 @@ cgroup_fj_function_net_prio cgroup_fj_function.sh net_prio
 cgroup_fj_function_hugetlb cgroup_fj_function.sh hugetlb
 
 # Stress test for debug cgroup
-cgroup_fj_stress_debug_2_2_none cgroup_fj_stress.sh debug 2 2 none
-cgroup_fj_stress_debug_3_3_none cgroup_fj_stress.sh debug 3 3 none
-cgroup_fj_stress_debug_4_4_none cgroup_fj_stress.sh debug 4 4 none
-cgroup_fj_stress_debug_2_9_none cgroup_fj_stress.sh debug 2 9 none
-cgroup_fj_stress_debug_10_3_none cgroup_fj_stress.sh debug 10 3 none
-cgroup_fj_stress_debug_1_200_none cgroup_fj_stress.sh debug 1 200 none
-cgroup_fj_stress_debug_200_1_none cgroup_fj_stress.sh debug 200 1 none
-
-cgroup_fj_stress_debug_2_2_one cgroup_fj_stress.sh debug 2 2 one
-cgroup_fj_stress_debug_3_3_one cgroup_fj_stress.sh debug 3 3 one
-cgroup_fj_stress_debug_4_4_one cgroup_fj_stress.sh debug 4 4 one
-cgroup_fj_stress_debug_2_9_one cgroup_fj_stress.sh debug 2 9 one
-cgroup_fj_stress_debug_10_3_one cgroup_fj_stress.sh debug 10 3 one
-cgroup_fj_stress_debug_1_200_one cgroup_fj_stress.sh debug 1 200 one
-cgroup_fj_stress_debug_200_1_one cgroup_fj_stress.sh debug 200 1 one
-
-cgroup_fj_stress_debug_2_2_each cgroup_fj_stress.sh debug 2 2 each
-cgroup_fj_stress_debug_3_3_each cgroup_fj_stress.sh debug 3 3 each
-cgroup_fj_stress_debug_4_4_each cgroup_fj_stress.sh debug 4 4 each
-cgroup_fj_stress_debug_2_9_each cgroup_fj_stress.sh debug 2 9 each
-cgroup_fj_stress_debug_10_3_each cgroup_fj_stress.sh debug 10 3 each
-cgroup_fj_stress_debug_1_200_each cgroup_fj_stress.sh debug 1 200 each
-cgroup_fj_stress_debug_200_1_each cgroup_fj_stress.sh debug 200 1 each
+#cgroup_fj_stress_debug_2_2_none cgroup_fj_stress.sh debug 2 2 none
+#cgroup_fj_stress_debug_3_3_none cgroup_fj_stress.sh debug 3 3 none
+#cgroup_fj_stress_debug_4_4_none cgroup_fj_stress.sh debug 4 4 none
+#cgroup_fj_stress_debug_2_9_none cgroup_fj_stress.sh debug 2 9 none
+#cgroup_fj_stress_debug_10_3_none cgroup_fj_stress.sh debug 10 3 none
+#cgroup_fj_stress_debug_1_200_none cgroup_fj_stress.sh debug 1 200 none
+#cgroup_fj_stress_debug_200_1_none cgroup_fj_stress.sh debug 200 1 none
+
+#cgroup_fj_stress_debug_2_2_one cgroup_fj_stress.sh debug 2 2 one
+#cgroup_fj_stress_debug_3_3_one cgroup_fj_stress.sh debug 3 3 one
+#cgroup_fj_stress_debug_4_4_one cgroup_fj_stress.sh debug 4 4 one
+#cgroup_fj_stress_debug_2_9_one cgroup_fj_stress.sh debug 2 9 one
+#cgroup_fj_stress_debug_10_3_one cgroup_fj_stress.sh debug 10 3 one
+#cgroup_fj_stress_debug_1_200_one cgroup_fj_stress.sh debug 1 200 one
+#cgroup_fj_stress_debug_200_1_one cgroup_fj_stress.sh debug 200 1 one
+
+#cgroup_fj_stress_debug_2_2_each cgroup_fj_stress.sh debug 2 2 each
+#cgroup_fj_stress_debug_3_3_each cgroup_fj_stress.sh debug 3 3 each
+#cgroup_fj_stress_debug_4_4_each cgroup_fj_stress.sh debug 4 4 each
+#cgroup_fj_stress_debug_2_9_each cgroup_fj_stress.sh debug 2 9 each
+#cgroup_fj_stress_debug_10_3_each cgroup_fj_stress.sh debug 10 3 each
+#cgroup_fj_stress_debug_1_200_each cgroup_fj_stress.sh debug 1 200 each
+#cgroup_fj_stress_debug_200_1_each cgroup_fj_stress.sh debug 200 1 each
 
 # Stress test for cpuset cgroup
-cgroup_fj_stress_cpuset_2_2_none cgroup_fj_stress.sh cpuset 2 2 none
-cgroup_fj_stress_cpuset_3_3_none cgroup_fj_stress.sh cpuset 3 3 none
-cgroup_fj_stress_cpuset_4_4_none cgroup_fj_stress.sh cpuset 4 4 none
-cgroup_fj_stress_cpuset_2_9_none cgroup_fj_stress.sh cpuset 2 9 none
-cgroup_fj_stress_cpuset_10_3_none cgroup_fj_stress.sh cpuset 10 3 none
-cgroup_fj_stress_cpuset_1_200_none cgroup_fj_stress.sh cpuset 1 200 none
-cgroup_fj_stress_cpuset_200_1_none cgroup_fj_stress.sh cpuset 200 1 none
-
-cgroup_fj_stress_cpuset_2_2_one cgroup_fj_stress.sh cpuset 2 2 one
-cgroup_fj_stress_cpuset_3_3_one cgroup_fj_stress.sh cpuset 3 3 one
-cgroup_fj_stress_cpuset_4_4_one cgroup_fj_stress.sh cpuset 4 4 one
-cgroup_fj_stress_cpuset_2_9_one cgroup_fj_stress.sh cpuset 2 9 one
-cgroup_fj_stress_cpuset_10_3_one cgroup_fj_stress.sh cpuset 10 3 one
-cgroup_fj_stress_cpuset_1_200_one cgroup_fj_stress.sh cpuset 1 200 one
-cgroup_fj_stress_cpuset_200_1_one cgroup_fj_stress.sh cpuset 200 1 one
-
-cgroup_fj_stress_cpuset_2_2_each cgroup_fj_stress.sh cpuset 2 2 each
-cgroup_fj_stress_cpuset_3_3_each cgroup_fj_stress.sh cpuset 3 3 each
-cgroup_fj_stress_cpuset_4_4_each cgroup_fj_stress.sh cpuset 4 4 each
-cgroup_fj_stress_cpuset_2_9_each cgroup_fj_stress.sh cpuset 2 9 each
-cgroup_fj_stress_cpuset_10_3_each cgroup_fj_stress.sh cpuset 10 3 each
-cgroup_fj_stress_cpuset_1_200_each cgroup_fj_stress.sh cpuset 1 200 each
-cgroup_fj_stress_cpuset_200_1_each cgroup_fj_stress.sh cpuset 200 1 each
+#cgroup_fj_stress_cpuset_2_2_none cgroup_fj_stress.sh cpuset 2 2 none
+#cgroup_fj_stress_cpuset_3_3_none cgroup_fj_stress.sh cpuset 3 3 none
+#cgroup_fj_stress_cpuset_4_4_none cgroup_fj_stress.sh cpuset 4 4 none
+#cgroup_fj_stress_cpuset_2_9_none cgroup_fj_stress.sh cpuset 2 9 none
+#cgroup_fj_stress_cpuset_10_3_none cgroup_fj_stress.sh cpuset 10 3 none
+#cgroup_fj_stress_cpuset_1_200_none cgroup_fj_stress.sh cpuset 1 200 none
+#cgroup_fj_stress_cpuset_200_1_none cgroup_fj_stress.sh cpuset 200 1 none
+
+#cgroup_fj_stress_cpuset_2_2_one cgroup_fj_stress.sh cpuset 2 2 one
+#cgroup_fj_stress_cpuset_3_3_one cgroup_fj_stress.sh cpuset 3 3 one
+#cgroup_fj_stress_cpuset_4_4_one cgroup_fj_stress.sh cpuset 4 4 one
+#cgroup_fj_stress_cpuset_2_9_one cgroup_fj_stress.sh cpuset 2 9 one
+#cgroup_fj_stress_cpuset_10_3_one cgroup_fj_stress.sh cpuset 10 3 one
+#cgroup_fj_stress_cpuset_1_200_one cgroup_fj_stress.sh cpuset 1 200 one
+#cgroup_fj_stress_cpuset_200_1_one cgroup_fj_stress.sh cpuset 200 1 one
+
+#cgroup_fj_stress_cpuset_2_2_each cgroup_fj_stress.sh cpuset 2 2 each
+#cgroup_fj_stress_cpuset_3_3_each cgroup_fj_stress.sh cpuset 3 3 each
+#cgroup_fj_stress_cpuset_4_4_each cgroup_fj_stress.sh cpuset 4 4 each
+#cgroup_fj_stress_cpuset_2_9_each cgroup_fj_stress.sh cpuset 2 9 each
+#cgroup_fj_stress_cpuset_10_3_each cgroup_fj_stress.sh cpuset 10 3 each
+#cgroup_fj_stress_cpuset_1_200_each cgroup_fj_stress.sh cpuset 1 200 each
+#cgroup_fj_stress_cpuset_200_1_each cgroup_fj_stress.sh cpuset 200 1 each
 
 # Stress test for cpu cgroup
-cgroup_fj_stress_cpu_2_2_none cgroup_fj_stress.sh cpu 2 2 none
-cgroup_fj_stress_cpu_3_3_none cgroup_fj_stress.sh cpu 3 3 none
-cgroup_fj_stress_cpu_4_4_none cgroup_fj_stress.sh cpu 4 4 none
-cgroup_fj_stress_cpu_2_9_none cgroup_fj_stress.sh cpu 2 9 none
-cgroup_fj_stress_cpu_10_3_none cgroup_fj_stress.sh cpu 10 3 none
-cgroup_fj_stress_cpu_1_200_none cgroup_fj_stress.sh cpu 1 200 none
-cgroup_fj_stress_cpu_200_1_none cgroup_fj_stress.sh cpu 200 1 none
-
-cgroup_fj_stress_cpu_2_2_one cgroup_fj_stress.sh cpu 2 2 one
-cgroup_fj_stress_cpu_3_3_one cgroup_fj_stress.sh cpu 3 3 one
-cgroup_fj_stress_cpu_4_4_one cgroup_fj_stress.sh cpu 4 4 one
-cgroup_fj_stress_cpu_2_9_one cgroup_fj_stress.sh cpu 2 9 one
-cgroup_fj_stress_cpu_10_3_one cgroup_fj_stress.sh cpu 10 3 one
-cgroup_fj_stress_cpu_1_200_one cgroup_fj_stress.sh cpu 1 200 one
-cgroup_fj_stress_cpu_200_1_one cgroup_fj_stress.sh cpu 200 1 one
-
-cgroup_fj_stress_cpu_2_2_each cgroup_fj_stress.sh cpu 2 2 each
-cgroup_fj_stress_cpu_3_3_each cgroup_fj_stress.sh cpu 3 3 each
-cgroup_fj_stress_cpu_4_4_each cgroup_fj_stress.sh cpu 4 4 each
-cgroup_fj_stress_cpu_2_9_each cgroup_fj_stress.sh cpu 2 9 each
-cgroup_fj_stress_cpu_10_3_each cgroup_fj_stress.sh cpu 10 3 each
-cgroup_fj_stress_cpu_1_200_each cgroup_fj_stress.sh cpu 1 200 each
-cgroup_fj_stress_cpu_200_1_each cgroup_fj_stress.sh cpu 200 1 each
+#cgroup_fj_stress_cpu_2_2_none cgroup_fj_stress.sh cpu 2 2 none
+#cgroup_fj_stress_cpu_3_3_none cgroup_fj_stress.sh cpu 3 3 none
+#cgroup_fj_stress_cpu_4_4_none cgroup_fj_stress.sh cpu 4 4 none
+#cgroup_fj_stress_cpu_2_9_none cgroup_fj_stress.sh cpu 2 9 none
+#cgroup_fj_stress_cpu_10_3_none cgroup_fj_stress.sh cpu 10 3 none
+#cgroup_fj_stress_cpu_1_200_none cgroup_fj_stress.sh cpu 1 200 none
+#cgroup_fj_stress_cpu_200_1_none cgroup_fj_stress.sh cpu 200 1 none
+
+#cgroup_fj_stress_cpu_2_2_one cgroup_fj_stress.sh cpu 2 2 one
+#cgroup_fj_stress_cpu_3_3_one cgroup_fj_stress.sh cpu 3 3 one
+#cgroup_fj_stress_cpu_4_4_one cgroup_fj_stress.sh cpu 4 4 one
+#cgroup_fj_stress_cpu_2_9_one cgroup_fj_stress.sh cpu 2 9 one
+#cgroup_fj_stress_cpu_10_3_one cgroup_fj_stress.sh cpu 10 3 one
+#cgroup_fj_stress_cpu_1_200_one cgroup_fj_stress.sh cpu 1 200 one
+#cgroup_fj_stress_cpu_200_1_one cgroup_fj_stress.sh cpu 200 1 one
+
+#cgroup_fj_stress_cpu_2_2_each cgroup_fj_stress.sh cpu 2 2 each
+#cgroup_fj_stress_cpu_3_3_each cgroup_fj_stress.sh cpu 3 3 each
+#cgroup_fj_stress_cpu_4_4_each cgroup_fj_stress.sh cpu 4 4 each
+#cgroup_fj_stress_cpu_2_9_each cgroup_fj_stress.sh cpu 2 9 each
+#cgroup_fj_stress_cpu_10_3_each cgroup_fj_stress.sh cpu 10 3 each
+#cgroup_fj_stress_cpu_1_200_each cgroup_fj_stress.sh cpu 1 200 each
+#cgroup_fj_stress_cpu_200_1_each cgroup_fj_stress.sh cpu 200 1 each
 
 # Stress test for cpuacct cgroup
-cgroup_fj_stress_cpuacct_2_2_none cgroup_fj_stress.sh cpuacct 2 2 none
-cgroup_fj_stress_cpuacct_3_3_none cgroup_fj_stress.sh cpuacct 3 3 none
-cgroup_fj_stress_cpuacct_4_4_none cgroup_fj_stress.sh cpuacct 4 4 none
-cgroup_fj_stress_cpuacct_2_9_none cgroup_fj_stress.sh cpuacct 2 9 none
-cgroup_fj_stress_cpuacct_10_3_none cgroup_fj_stress.sh cpuacct 10 3 none
-cgroup_fj_stress_cpuacct_1_200_none cgroup_fj_stress.sh cpuacct 1 200 none
-cgroup_fj_stress_cpuacct_200_1_none cgroup_fj_stress.sh cpuacct 200 1 none
-
-cgroup_fj_stress_cpuacct_2_2_one cgroup_fj_stress.sh cpuacct 2 2 one
-cgroup_fj_stress_cpuacct_3_3_one cgroup_fj_stress.sh cpuacct 3 3 one
-cgroup_fj_stress_cpuacct_4_4_one cgroup_fj_stress.sh cpuacct 4 4 one
-cgroup_fj_stress_cpuacct_2_9_one cgroup_fj_stress.sh cpuacct 2 9 one
-cgroup_fj_stress_cpuacct_10_3_one cgroup_fj_stress.sh cpuacct 10 3 one
-cgroup_fj_stress_cpuacct_1_200_one cgroup_fj_stress.sh cpuacct 1 200 one
-cgroup_fj_stress_cpuacct_200_1_one cgroup_fj_stress.sh cpuacct 200 1 one
-
-cgroup_fj_stress_cpuacct_2_2_each cgroup_fj_stress.sh cpuacct 2 2 each
-cgroup_fj_stress_cpuacct_3_3_each cgroup_fj_stress.sh cpuacct 3 3 each
-cgroup_fj_stress_cpuacct_4_4_each cgroup_fj_stress.sh cpuacct 4 4 each
-cgroup_fj_stress_cpuacct_2_9_each cgroup_fj_stress.sh cpuacct 2 9 each
-cgroup_fj_stress_cpuacct_10_3_each cgroup_fj_stress.sh cpuacct 10 3 each
-cgroup_fj_stress_cpuacct_1_200_each cgroup_fj_stress.sh cpuacct 1 200 each
-cgroup_fj_stress_cpuacct_200_1_each cgroup_fj_stress.sh cpuacct 200 1 each
+#cgroup_fj_stress_cpuacct_2_2_none cgroup_fj_stress.sh cpuacct 2 2 none
+#cgroup_fj_stress_cpuacct_3_3_none cgroup_fj_stress.sh cpuacct 3 3 none
+#cgroup_fj_stress_cpuacct_4_4_none cgroup_fj_stress.sh cpuacct 4 4 none
+#cgroup_fj_stress_cpuacct_2_9_none cgroup_fj_stress.sh cpuacct 2 9 none
+#cgroup_fj_stress_cpuacct_10_3_none cgroup_fj_stress.sh cpuacct 10 3 none
+#cgroup_fj_stress_cpuacct_1_200_none cgroup_fj_stress.sh cpuacct 1 200 none
+#cgroup_fj_stress_cpuacct_200_1_none cgroup_fj_stress.sh cpuacct 200 1 none
+
+#cgroup_fj_stress_cpuacct_2_2_one cgroup_fj_stress.sh cpuacct 2 2 one
+#cgroup_fj_stress_cpuacct_3_3_one cgroup_fj_stress.sh cpuacct 3 3 one
+#cgroup_fj_stress_cpuacct_4_4_one cgroup_fj_stress.sh cpuacct 4 4 one
+#cgroup_fj_stress_cpuacct_2_9_one cgroup_fj_stress.sh cpuacct 2 9 one
+#cgroup_fj_stress_cpuacct_10_3_one cgroup_fj_stress.sh cpuacct 10 3 one
+#cgroup_fj_stress_cpuacct_1_200_one cgroup_fj_stress.sh cpuacct 1 200 one
+#cgroup_fj_stress_cpuacct_200_1_one cgroup_fj_stress.sh cpuacct 200 1 one
+
+#cgroup_fj_stress_cpuacct_2_2_each cgroup_fj_stress.sh cpuacct 2 2 each
+#cgroup_fj_stress_cpuacct_3_3_each cgroup_fj_stress.sh cpuacct 3 3 each
+#cgroup_fj_stress_cpuacct_4_4_each cgroup_fj_stress.sh cpuacct 4 4 each
+#cgroup_fj_stress_cpuacct_2_9_each cgroup_fj_stress.sh cpuacct 2 9 each
+#cgroup_fj_stress_cpuacct_10_3_each cgroup_fj_stress.sh cpuacct 10 3 each
+#cgroup_fj_stress_cpuacct_1_200_each cgroup_fj_stress.sh cpuacct 1 200 each
+#cgroup_fj_stress_cpuacct_200_1_each cgroup_fj_stress.sh cpuacct 200 1 each
 
 # Stress test for memory cgroup
-cgroup_fj_stress_memory_2_2_none cgroup_fj_stress.sh memory 2 2 none
-cgroup_fj_stress_memory_3_3_none cgroup_fj_stress.sh memory 3 3 none
-cgroup_fj_stress_memory_4_4_none cgroup_fj_stress.sh memory 4 4 none
-cgroup_fj_stress_memory_2_9_none cgroup_fj_stress.sh memory 2 9 none
-cgroup_fj_stress_memory_10_3_none cgroup_fj_stress.sh memory 10 3 none
-cgroup_fj_stress_memory_1_200_none cgroup_fj_stress.sh memory 1 200 none
-cgroup_fj_stress_memory_200_1_none cgroup_fj_stress.sh memory 200 1 none
-
-cgroup_fj_stress_memory_2_2_one cgroup_fj_stress.sh memory 2 2 one
-cgroup_fj_stress_memory_3_3_one cgroup_fj_stress.sh memory 3 3 one
-cgroup_fj_stress_memory_4_4_one cgroup_fj_stress.sh memory 4 4 one
-cgroup_fj_stress_memory_2_9_one cgroup_fj_stress.sh memory 2 9 one
-cgroup_fj_stress_memory_10_3_one cgroup_fj_stress.sh memory 10 3 one
-cgroup_fj_stress_memory_1_200_one cgroup_fj_stress.sh memory 1 200 one
-cgroup_fj_stress_memory_200_1_one cgroup_fj_stress.sh memory 200 1 one
-
-cgroup_fj_stress_memory_2_2_each cgroup_fj_stress.sh memory 2 2 each
-cgroup_fj_stress_memory_3_3_each cgroup_fj_stress.sh memory 3 3 each
-cgroup_fj_stress_memory_4_4_each cgroup_fj_stress.sh memory 4 4 each
-cgroup_fj_stress_memory_2_9_each cgroup_fj_stress.sh memory 2 9 each
-cgroup_fj_stress_memory_10_3_each cgroup_fj_stress.sh memory 10 3 each
-cgroup_fj_stress_memory_1_200_each cgroup_fj_stress.sh memory 1 200 each
-cgroup_fj_stress_memory_200_1_each cgroup_fj_stress.sh memory 200 1 each
+#cgroup_fj_stress_memory_2_2_none cgroup_fj_stress.sh memory 2 2 none
+#cgroup_fj_stress_memory_3_3_none cgroup_fj_stress.sh memory 3 3 none
+#cgroup_fj_stress_memory_4_4_none cgroup_fj_stress.sh memory 4 4 none
+#cgroup_fj_stress_memory_2_9_none cgroup_fj_stress.sh memory 2 9 none
+#cgroup_fj_stress_memory_10_3_none cgroup_fj_stress.sh memory 10 3 none
+#cgroup_fj_stress_memory_1_200_none cgroup_fj_stress.sh memory 1 200 none
+#cgroup_fj_stress_memory_200_1_none cgroup_fj_stress.sh memory 200 1 none
+
+#cgroup_fj_stress_memory_2_2_one cgroup_fj_stress.sh memory 2 2 one
+#cgroup_fj_stress_memory_3_3_one cgroup_fj_stress.sh memory 3 3 one
+#cgroup_fj_stress_memory_4_4_one cgroup_fj_stress.sh memory 4 4 one
+#cgroup_fj_stress_memory_2_9_one cgroup_fj_stress.sh memory 2 9 one
+#cgroup_fj_stress_memory_10_3_one cgroup_fj_stress.sh memory 10 3 one
+#cgroup_fj_stress_memory_1_200_one cgroup_fj_stress.sh memory 1 200 one
+#cgroup_fj_stress_memory_200_1_one cgroup_fj_stress.sh memory 200 1 one
+
+#cgroup_fj_stress_memory_2_2_each cgroup_fj_stress.sh memory 2 2 each
+#cgroup_fj_stress_memory_3_3_each cgroup_fj_stress.sh memory 3 3 each
+#cgroup_fj_stress_memory_4_4_each cgroup_fj_stress.sh memory 4 4 each
+#cgroup_fj_stress_memory_2_9_each cgroup_fj_stress.sh memory 2 9 each
+#cgroup_fj_stress_memory_10_3_each cgroup_fj_stress.sh memory 10 3 each
+#cgroup_fj_stress_memory_1_200_each cgroup_fj_stress.sh memory 1 200 each
+#cgroup_fj_stress_memory_200_1_each cgroup_fj_stress.sh memory 200 1 each
 
 # Stress test for freezer cgroup
-cgroup_fj_stress_freezer_2_2_none cgroup_fj_stress.sh freezer 2 2 none
-cgroup_fj_stress_freezer_3_3_none cgroup_fj_stress.sh freezer 3 3 none
-cgroup_fj_stress_freezer_4_4_none cgroup_fj_stress.sh freezer 4 4 none
-cgroup_fj_stress_freezer_2_9_none cgroup_fj_stress.sh freezer 2 9 none
-cgroup_fj_stress_freezer_10_3_none cgroup_fj_stress.sh freezer 10 3 none
-cgroup_fj_stress_freezer_1_200_none cgroup_fj_stress.sh freezer 1 200 none
-cgroup_fj_stress_freezer_200_1_none cgroup_fj_stress.sh freezer 200 1 none
-
-cgroup_fj_stress_freezer_2_2_one cgroup_fj_stress.sh freezer 2 2 one
-cgroup_fj_stress_freezer_3_3_one cgroup_fj_stress.sh freezer 3 3 one
-cgroup_fj_stress_freezer_4_4_one cgroup_fj_stress.sh freezer 4 4 one
-cgroup_fj_stress_freezer_2_9_one cgroup_fj_stress.sh freezer 2 9 one
-cgroup_fj_stress_freezer_10_3_one cgroup_fj_stress.sh freezer 10 3 one
-cgroup_fj_stress_freezer_1_200_one cgroup_fj_stress.sh freezer 1 200 one
-cgroup_fj_stress_freezer_200_1_one cgroup_fj_stress.sh freezer 200 1 one
-
-cgroup_fj_stress_freezer_2_2_each cgroup_fj_stress.sh freezer 2 2 each
-cgroup_fj_stress_freezer_3_3_each cgroup_fj_stress.sh freezer 3 3 each
-cgroup_fj_stress_freezer_4_4_each cgroup_fj_stress.sh freezer 4 4 each
-cgroup_fj_stress_freezer_2_9_each cgroup_fj_stress.sh freezer 2 9 each
-cgroup_fj_stress_freezer_10_3_each cgroup_fj_stress.sh freezer 10 3 each
-cgroup_fj_stress_freezer_1_200_each cgroup_fj_stress.sh freezer 1 200 each
-cgroup_fj_stress_freezer_200_1_each cgroup_fj_stress.sh freezer 200 1 each
+#cgroup_fj_stress_freezer_2_2_none cgroup_fj_stress.sh freezer 2 2 none
+#cgroup_fj_stress_freezer_3_3_none cgroup_fj_stress.sh freezer 3 3 none
+#cgroup_fj_stress_freezer_4_4_none cgroup_fj_stress.sh freezer 4 4 none
+#cgroup_fj_stress_freezer_2_9_none cgroup_fj_stress.sh freezer 2 9 none
+#cgroup_fj_stress_freezer_10_3_none cgroup_fj_stress.sh freezer 10 3 none
+#cgroup_fj_stress_freezer_1_200_none cgroup_fj_stress.sh freezer 1 200 none
+#cgroup_fj_stress_freezer_200_1_none cgroup_fj_stress.sh freezer 200 1 none
+
+#cgroup_fj_stress_freezer_2_2_one cgroup_fj_stress.sh freezer 2 2 one
+#cgroup_fj_stress_freezer_3_3_one cgroup_fj_stress.sh freezer 3 3 one
+#cgroup_fj_stress_freezer_4_4_one cgroup_fj_stress.sh freezer 4 4 one
+#cgroup_fj_stress_freezer_2_9_one cgroup_fj_stress.sh freezer 2 9 one
+#cgroup_fj_stress_freezer_10_3_one cgroup_fj_stress.sh freezer 10 3 one
+#cgroup_fj_stress_freezer_1_200_one cgroup_fj_stress.sh freezer 1 200 one
+#cgroup_fj_stress_freezer_200_1_one cgroup_fj_stress.sh freezer 200 1 one
+
+#cgroup_fj_stress_freezer_2_2_each cgroup_fj_stress.sh freezer 2 2 each
+#cgroup_fj_stress_freezer_3_3_each cgroup_fj_stress.sh freezer 3 3 each
+#cgroup_fj_stress_freezer_4_4_each cgroup_fj_stress.sh freezer 4 4 each
+#cgroup_fj_stress_freezer_2_9_each cgroup_fj_stress.sh freezer 2 9 each
+#cgroup_fj_stress_freezer_10_3_each cgroup_fj_stress.sh freezer 10 3 each
+#cgroup_fj_stress_freezer_1_200_each cgroup_fj_stress.sh freezer 1 200 each
+#cgroup_fj_stress_freezer_200_1_each cgroup_fj_stress.sh freezer 200 1 each
 
 # Stress test for devices cgroup
-cgroup_fj_stress_devices_2_2_none cgroup_fj_stress.sh devices 2 2 none
-cgroup_fj_stress_devices_3_3_none cgroup_fj_stress.sh devices 3 3 none
-cgroup_fj_stress_devices_4_4_none cgroup_fj_stress.sh devices 4 4 none
-cgroup_fj_stress_devices_2_9_none cgroup_fj_stress.sh devices 2 9 none
-cgroup_fj_stress_devices_10_3_none cgroup_fj_stress.sh devices 10 3 none
-cgroup_fj_stress_devices_1_200_none cgroup_fj_stress.sh devices 1 200 none
-cgroup_fj_stress_devices_200_1_none cgroup_fj_stress.sh devices 200 1 none
-
-cgroup_fj_stress_devices_2_2_one cgroup_fj_stress.sh devices 2 2 one
-cgroup_fj_stress_devices_3_3_one cgroup_fj_stress.sh devices 3 3 one
-cgroup_fj_stress_devices_4_4_one cgroup_fj_stress.sh devices 4 4 one
-cgroup_fj_stress_devices_2_9_one cgroup_fj_stress.sh devices 2 9 one
-cgroup_fj_stress_devices_10_3_one cgroup_fj_stress.sh devices 10 3 one
-cgroup_fj_stress_devices_1_200_one cgroup_fj_stress.sh devices 1 200 one
-cgroup_fj_stress_devices_200_1_one cgroup_fj_stress.sh devices 200 1 one
-
-cgroup_fj_stress_devices_2_2_each cgroup_fj_stress.sh devices 2 2 each
-cgroup_fj_stress_devices_3_3_each cgroup_fj_stress.sh devices 3 3 each
-cgroup_fj_stress_devices_4_4_each cgroup_fj_stress.sh devices 4 4 each
-cgroup_fj_stress_devices_2_9_each cgroup_fj_stress.sh devices 2 9 each
-cgroup_fj_stress_devices_10_3_each cgroup_fj_stress.sh devices 10 3 each
-cgroup_fj_stress_devices_1_200_each cgroup_fj_stress.sh devices 1 200 each
-cgroup_fj_stress_devices_200_1_each cgroup_fj_stress.sh devices 200 1 each
+#cgroup_fj_stress_devices_2_2_none cgroup_fj_stress.sh devices 2 2 none
+#cgroup_fj_stress_devices_3_3_none cgroup_fj_stress.sh devices 3 3 none
+#cgroup_fj_stress_devices_4_4_none cgroup_fj_stress.sh devices 4 4 none
+#cgroup_fj_stress_devices_2_9_none cgroup_fj_stress.sh devices 2 9 none
+#cgroup_fj_stress_devices_10_3_none cgroup_fj_stress.sh devices 10 3 none
+#cgroup_fj_stress_devices_1_200_none cgroup_fj_stress.sh devices 1 200 none
+#cgroup_fj_stress_devices_200_1_none cgroup_fj_stress.sh devices 200 1 none
+
+#cgroup_fj_stress_devices_2_2_one cgroup_fj_stress.sh devices 2 2 one
+#cgroup_fj_stress_devices_3_3_one cgroup_fj_stress.sh devices 3 3 one
+#cgroup_fj_stress_devices_4_4_one cgroup_fj_stress.sh devices 4 4 one
+#cgroup_fj_stress_devices_2_9_one cgroup_fj_stress.sh devices 2 9 one
+#cgroup_fj_stress_devices_10_3_one cgroup_fj_stress.sh devices 10 3 one
+#cgroup_fj_stress_devices_1_200_one cgroup_fj_stress.sh devices 1 200 one
+#cgroup_fj_stress_devices_200_1_one cgroup_fj_stress.sh devices 200 1 one
+
+#cgroup_fj_stress_devices_2_2_each cgroup_fj_stress.sh devices 2 2 each
+#cgroup_fj_stress_devices_3_3_each cgroup_fj_stress.sh devices 3 3 each
+#cgroup_fj_stress_devices_4_4_each cgroup_fj_stress.sh devices 4 4 each
+#cgroup_fj_stress_devices_2_9_each cgroup_fj_stress.sh devices 2 9 each
+#cgroup_fj_stress_devices_10_3_each cgroup_fj_stress.sh devices 10 3 each
+#cgroup_fj_stress_devices_1_200_each cgroup_fj_stress.sh devices 1 200 each
+#cgroup_fj_stress_devices_200_1_each cgroup_fj_stress.sh devices 200 1 each
 
 # Stress test for blkio cgroup
-cgroup_fj_stress_blkio_2_2_none cgroup_fj_stress.sh blkio 2 2 none
-cgroup_fj_stress_blkio_3_3_none cgroup_fj_stress.sh blkio 3 3 none
-cgroup_fj_stress_blkio_4_4_none cgroup_fj_stress.sh blkio 4 4 none
-cgroup_fj_stress_blkio_2_9_none cgroup_fj_stress.sh blkio 2 9 none
-cgroup_fj_stress_blkio_10_3_none cgroup_fj_stress.sh blkio 10 3 none
-cgroup_fj_stress_blkio_1_200_none cgroup_fj_stress.sh blkio 1 200 none
-cgroup_fj_stress_blkio_200_1_none cgroup_fj_stress.sh blkio 200 1 none
-
-cgroup_fj_stress_blkio_2_2_one cgroup_fj_stress.sh blkio 2 2 one
-cgroup_fj_stress_blkio_3_3_one cgroup_fj_stress.sh blkio 3 3 one
-cgroup_fj_stress_blkio_4_4_one cgroup_fj_stress.sh blkio 4 4 one
-cgroup_fj_stress_blkio_2_9_one cgroup_fj_stress.sh blkio 2 9 one
-cgroup_fj_stress_blkio_10_3_one cgroup_fj_stress.sh blkio 10 3 one
-cgroup_fj_stress_blkio_1_200_one cgroup_fj_stress.sh blkio 1 200 one
-cgroup_fj_stress_blkio_200_1_one cgroup_fj_stress.sh blkio 200 1 one
-
-cgroup_fj_stress_blkio_2_2_each cgroup_fj_stress.sh blkio 2 2 each
-cgroup_fj_stress_blkio_3_3_each cgroup_fj_stress.sh blkio 3 3 each
-cgroup_fj_stress_blkio_4_4_each cgroup_fj_stress.sh blkio 4 4 each
-cgroup_fj_stress_blkio_2_9_each cgroup_fj_stress.sh blkio 2 9 each
-cgroup_fj_stress_blkio_10_3_each cgroup_fj_stress.sh blkio 10 3 each
-cgroup_fj_stress_blkio_1_200_each cgroup_fj_stress.sh blkio 1 200 each
-cgroup_fj_stress_blkio_200_1_each cgroup_fj_stress.sh blkio 200 1 each
+#cgroup_fj_stress_blkio_2_2_none cgroup_fj_stress.sh blkio 2 2 none
+#cgroup_fj_stress_blkio_3_3_none cgroup_fj_stress.sh blkio 3 3 none
+#cgroup_fj_stress_blkio_4_4_none cgroup_fj_stress.sh blkio 4 4 none
+#cgroup_fj_stress_blkio_2_9_none cgroup_fj_stress.sh blkio 2 9 none
+#cgroup_fj_stress_blkio_10_3_none cgroup_fj_stress.sh blkio 10 3 none
+#cgroup_fj_stress_blkio_1_200_none cgroup_fj_stress.sh blkio 1 200 none
+#cgroup_fj_stress_blkio_200_1_none cgroup_fj_stress.sh blkio 200 1 none
+
+#cgroup_fj_stress_blkio_2_2_one cgroup_fj_stress.sh blkio 2 2 one
+#cgroup_fj_stress_blkio_3_3_one cgroup_fj_stress.sh blkio 3 3 one
+#cgroup_fj_stress_blkio_4_4_one cgroup_fj_stress.sh blkio 4 4 one
+#cgroup_fj_stress_blkio_2_9_one cgroup_fj_stress.sh blkio 2 9 one
+#cgroup_fj_stress_blkio_10_3_one cgroup_fj_stress.sh blkio 10 3 one
+#cgroup_fj_stress_blkio_1_200_one cgroup_fj_stress.sh blkio 1 200 one
+#cgroup_fj_stress_blkio_200_1_one cgroup_fj_stress.sh blkio 200 1 one
+
+#cgroup_fj_stress_blkio_2_2_each cgroup_fj_stress.sh blkio 2 2 each
+#cgroup_fj_stress_blkio_3_3_each cgroup_fj_stress.sh blkio 3 3 each
+#cgroup_fj_stress_blkio_4_4_each cgroup_fj_stress.sh blkio 4 4 each
+#cgroup_fj_stress_blkio_2_9_each cgroup_fj_stress.sh blkio 2 9 each
+#cgroup_fj_stress_blkio_10_3_each cgroup_fj_stress.sh blkio 10 3 each
+#cgroup_fj_stress_blkio_1_200_each cgroup_fj_stress.sh blkio 1 200 each
+#cgroup_fj_stress_blkio_200_1_each cgroup_fj_stress.sh blkio 200 1 each
 
 # Stress test for net_cls cgroup
-cgroup_fj_stress_net_cls_2_2_none cgroup_fj_stress.sh net_cls 2 2 none
-cgroup_fj_stress_net_cls_3_3_none cgroup_fj_stress.sh net_cls 3 3 none
-cgroup_fj_stress_net_cls_4_4_none cgroup_fj_stress.sh net_cls 4 4 none
-cgroup_fj_stress_net_cls_2_9_none cgroup_fj_stress.sh net_cls 2 9 none
-cgroup_fj_stress_net_cls_10_3_none cgroup_fj_stress.sh net_cls 10 3 none
-cgroup_fj_stress_net_cls_1_200_none cgroup_fj_stress.sh net_cls 1 200 none
-cgroup_fj_stress_net_cls_200_1_none cgroup_fj_stress.sh net_cls 200 1 none
-
-cgroup_fj_stress_net_cls_2_2_one cgroup_fj_stress.sh net_cls 2 2 one
-cgroup_fj_stress_net_cls_3_3_one cgroup_fj_stress.sh net_cls 3 3 one
-cgroup_fj_stress_net_cls_4_4_one cgroup_fj_stress.sh net_cls 4 4 one
-cgroup_fj_stress_net_cls_2_9_one cgroup_fj_stress.sh net_cls 2 9 one
-cgroup_fj_stress_net_cls_10_3_one cgroup_fj_stress.sh net_cls 10 3 one
-cgroup_fj_stress_net_cls_1_200_one cgroup_fj_stress.sh net_cls 1 200 one
-cgroup_fj_stress_net_cls_200_1_one cgroup_fj_stress.sh net_cls 200 1 one
-
-cgroup_fj_stress_net_cls_2_2_each cgroup_fj_stress.sh net_cls 2 2 each
-cgroup_fj_stress_net_cls_3_3_each cgroup_fj_stress.sh net_cls 3 3 each
-cgroup_fj_stress_net_cls_4_4_each cgroup_fj_stress.sh net_cls 4 4 each
-cgroup_fj_stress_net_cls_2_9_each cgroup_fj_stress.sh net_cls 2 9 each
-cgroup_fj_stress_net_cls_10_3_each cgroup_fj_stress.sh net_cls 10 3 each
-cgroup_fj_stress_net_cls_1_200_each cgroup_fj_stress.sh net_cls 1 200 each
-cgroup_fj_stress_net_cls_200_1_each cgroup_fj_stress.sh net_cls 200 1 each
+#cgroup_fj_stress_net_cls_2_2_none cgroup_fj_stress.sh net_cls 2 2 none
+#cgroup_fj_stress_net_cls_3_3_none cgroup_fj_stress.sh net_cls 3 3 none
+#cgroup_fj_stress_net_cls_4_4_none cgroup_fj_stress.sh net_cls 4 4 none
+#cgroup_fj_stress_net_cls_2_9_none cgroup_fj_stress.sh net_cls 2 9 none
+#cgroup_fj_stress_net_cls_10_3_none cgroup_fj_stress.sh net_cls 10 3 none
+#cgroup_fj_stress_net_cls_1_200_none cgroup_fj_stress.sh net_cls 1 200 none
+#cgroup_fj_stress_net_cls_200_1_none cgroup_fj_stress.sh net_cls 200 1 none
+
+#cgroup_fj_stress_net_cls_2_2_one cgroup_fj_stress.sh net_cls 2 2 one
+#cgroup_fj_stress_net_cls_3_3_one cgroup_fj_stress.sh net_cls 3 3 one
+#cgroup_fj_stress_net_cls_4_4_one cgroup_fj_stress.sh net_cls 4 4 one
+#cgroup_fj_stress_net_cls_2_9_one cgroup_fj_stress.sh net_cls 2 9 one
+#cgroup_fj_stress_net_cls_10_3_one cgroup_fj_stress.sh net_cls 10 3 one
+#cgroup_fj_stress_net_cls_1_200_one cgroup_fj_stress.sh net_cls 1 200 one
+#cgroup_fj_stress_net_cls_200_1_one cgroup_fj_stress.sh net_cls 200 1 one
+
+#cgroup_fj_stress_net_cls_2_2_each cgroup_fj_stress.sh net_cls 2 2 each
+#cgroup_fj_stress_net_cls_3_3_each cgroup_fj_stress.sh net_cls 3 3 each
+#cgroup_fj_stress_net_cls_4_4_each cgroup_fj_stress.sh net_cls 4 4 each
+#cgroup_fj_stress_net_cls_2_9_each cgroup_fj_stress.sh net_cls 2 9 each
+#cgroup_fj_stress_net_cls_10_3_each cgroup_fj_stress.sh net_cls 10 3 each
+#cgroup_fj_stress_net_cls_1_200_each cgroup_fj_stress.sh net_cls 1 200 each
+#cgroup_fj_stress_net_cls_200_1_each cgroup_fj_stress.sh net_cls 200 1 each
 
 # Stress test for perf_event cgroup
-cgroup_fj_stress_perf_event_2_2_none cgroup_fj_stress.sh perf_event 2 2 none
-cgroup_fj_stress_perf_event_3_3_none cgroup_fj_stress.sh perf_event 3 3 none
-cgroup_fj_stress_perf_event_4_4_none cgroup_fj_stress.sh perf_event 4 4 none
-cgroup_fj_stress_perf_event_2_9_none cgroup_fj_stress.sh perf_event 2 9 none
-cgroup_fj_stress_perf_event_10_3_none cgroup_fj_stress.sh perf_event 10 3 none
-cgroup_fj_stress_perf_event_1_200_none cgroup_fj_stress.sh perf_event 1 200 none
-cgroup_fj_stress_perf_event_200_1_none cgroup_fj_stress.sh perf_event 200 1 none
-
-cgroup_fj_stress_perf_event_2_2_one cgroup_fj_stress.sh perf_event 2 2 one
-cgroup_fj_stress_perf_event_3_3_one cgroup_fj_stress.sh perf_event 3 3 one
-cgroup_fj_stress_perf_event_4_4_one cgroup_fj_stress.sh perf_event 4 4 one
-cgroup_fj_stress_perf_event_2_9_one cgroup_fj_stress.sh perf_event 2 9 one
-cgroup_fj_stress_perf_event_10_3_one cgroup_fj_stress.sh perf_event 10 3 one
-cgroup_fj_stress_perf_event_1_200_one cgroup_fj_stress.sh perf_event 1 200 one
-cgroup_fj_stress_perf_event_200_1_one cgroup_fj_stress.sh perf_event 200 1 one
-
-cgroup_fj_stress_perf_event_2_2_each cgroup_fj_stress.sh perf_event 2 2 each
-cgroup_fj_stress_perf_event_3_3_each cgroup_fj_stress.sh perf_event 3 3 each
-cgroup_fj_stress_perf_event_4_4_each cgroup_fj_stress.sh perf_event 4 4 each
-cgroup_fj_stress_perf_event_2_9_each cgroup_fj_stress.sh perf_event 2 9 each
-cgroup_fj_stress_perf_event_10_3_each cgroup_fj_stress.sh perf_event 10 3 each
-cgroup_fj_stress_perf_event_1_200_each cgroup_fj_stress.sh perf_event 1 200 each
-cgroup_fj_stress_perf_event_200_1_each cgroup_fj_stress.sh perf_event 200 1 each
+#cgroup_fj_stress_perf_event_2_2_none cgroup_fj_stress.sh perf_event 2 2 none
+#cgroup_fj_stress_perf_event_3_3_none cgroup_fj_stress.sh perf_event 3 3 none
+#cgroup_fj_stress_perf_event_4_4_none cgroup_fj_stress.sh perf_event 4 4 none
+#cgroup_fj_stress_perf_event_2_9_none cgroup_fj_stress.sh perf_event 2 9 none
+#cgroup_fj_stress_perf_event_10_3_none cgroup_fj_stress.sh perf_event 10 3 none
+#cgroup_fj_stress_perf_event_1_200_none cgroup_fj_stress.sh perf_event 1 200 none
+#cgroup_fj_stress_perf_event_200_1_none cgroup_fj_stress.sh perf_event 200 1 none
+
+#cgroup_fj_stress_perf_event_2_2_one cgroup_fj_stress.sh perf_event 2 2 one
+#cgroup_fj_stress_perf_event_3_3_one cgroup_fj_stress.sh perf_event 3 3 one
+#cgroup_fj_stress_perf_event_4_4_one cgroup_fj_stress.sh perf_event 4 4 one
+#cgroup_fj_stress_perf_event_2_9_one cgroup_fj_stress.sh perf_event 2 9 one
+#cgroup_fj_stress_perf_event_10_3_one cgroup_fj_stress.sh perf_event 10 3 one
+#cgroup_fj_stress_perf_event_1_200_one cgroup_fj_stress.sh perf_event 1 200 one
+#cgroup_fj_stress_perf_event_200_1_one cgroup_fj_stress.sh perf_event 200 1 one
+
+#cgroup_fj_stress_perf_event_2_2_each cgroup_fj_stress.sh perf_event 2 2 each
+#cgroup_fj_stress_perf_event_3_3_each cgroup_fj_stress.sh perf_event 3 3 each
+#cgroup_fj_stress_perf_event_4_4_each cgroup_fj_stress.sh perf_event 4 4 each
+#cgroup_fj_stress_perf_event_2_9_each cgroup_fj_stress.sh perf_event 2 9 each
+#cgroup_fj_stress_perf_event_10_3_each cgroup_fj_stress.sh perf_event 10 3 each
+#cgroup_fj_stress_perf_event_1_200_each cgroup_fj_stress.sh perf_event 1 200 each
+#cgroup_fj_stress_perf_event_200_1_each cgroup_fj_stress.sh perf_event 200 1 each
 
 # Stress test for net_prio cgroup
-cgroup_fj_stress_net_prio_2_2_none cgroup_fj_stress.sh net_prio 2 2 none
-cgroup_fj_stress_net_prio_3_3_none cgroup_fj_stress.sh net_prio 3 3 none
-cgroup_fj_stress_net_prio_4_4_none cgroup_fj_stress.sh net_prio 4 4 none
-cgroup_fj_stress_net_prio_2_9_none cgroup_fj_stress.sh net_prio 2 9 none
-cgroup_fj_stress_net_prio_10_3_none cgroup_fj_stress.sh net_prio 10 3 none
-cgroup_fj_stress_net_prio_1_200_none cgroup_fj_stress.sh net_prio 1 200 none
-cgroup_fj_stress_net_prio_200_1_none cgroup_fj_stress.sh net_prio 200 1 none
-
-cgroup_fj_stress_net_prio_2_2_one cgroup_fj_stress.sh net_prio 2 2 one
-cgroup_fj_stress_net_prio_3_3_one cgroup_fj_stress.sh net_prio 3 3 one
-cgroup_fj_stress_net_prio_4_4_one cgroup_fj_stress.sh net_prio 4 4 one
-cgroup_fj_stress_net_prio_2_9_one cgroup_fj_stress.sh net_prio 2 9 one
-cgroup_fj_stress_net_prio_10_3_one cgroup_fj_stress.sh net_prio 10 3 one
-cgroup_fj_stress_net_prio_1_200_one cgroup_fj_stress.sh net_prio 1 200 one
-cgroup_fj_stress_net_prio_200_1_one cgroup_fj_stress.sh net_prio 200 1 one
-
-cgroup_fj_stress_net_prio_2_2_each cgroup_fj_stress.sh net_prio 2 2 each
-cgroup_fj_stress_net_prio_3_3_each cgroup_fj_stress.sh net_prio 3 3 each
-cgroup_fj_stress_net_prio_4_4_each cgroup_fj_stress.sh net_prio 4 4 each
-cgroup_fj_stress_net_prio_2_9_each cgroup_fj_stress.sh net_prio 2 9 each
-cgroup_fj_stress_net_prio_10_3_each cgroup_fj_stress.sh net_prio 10 3 each
-cgroup_fj_stress_net_prio_1_200_each cgroup_fj_stress.sh net_prio 1 200 each
-cgroup_fj_stress_net_prio_200_1_each cgroup_fj_stress.sh net_prio 200 1 each
+#cgroup_fj_stress_net_prio_2_2_none cgroup_fj_stress.sh net_prio 2 2 none
+#cgroup_fj_stress_net_prio_3_3_none cgroup_fj_stress.sh net_prio 3 3 none
+#cgroup_fj_stress_net_prio_4_4_none cgroup_fj_stress.sh net_prio 4 4 none
+#cgroup_fj_stress_net_prio_2_9_none cgroup_fj_stress.sh net_prio 2 9 none
+#cgroup_fj_stress_net_prio_10_3_none cgroup_fj_stress.sh net_prio 10 3 none
+#cgroup_fj_stress_net_prio_1_200_none cgroup_fj_stress.sh net_prio 1 200 none
+#cgroup_fj_stress_net_prio_200_1_none cgroup_fj_stress.sh net_prio 200 1 none
+
+#cgroup_fj_stress_net_prio_2_2_one cgroup_fj_stress.sh net_prio 2 2 one
+#cgroup_fj_stress_net_prio_3_3_one cgroup_fj_stress.sh net_prio 3 3 one
+#cgroup_fj_stress_net_prio_4_4_one cgroup_fj_stress.sh net_prio 4 4 one
+#cgroup_fj_stress_net_prio_2_9_one cgroup_fj_stress.sh net_prio 2 9 one
+#cgroup_fj_stress_net_prio_10_3_one cgroup_fj_stress.sh net_prio 10 3 one
+#cgroup_fj_stress_net_prio_1_200_one cgroup_fj_stress.sh net_prio 1 200 one
+#cgroup_fj_stress_net_prio_200_1_one cgroup_fj_stress.sh net_prio 200 1 one
+
+#cgroup_fj_stress_net_prio_2_2_each cgroup_fj_stress.sh net_prio 2 2 each
+#cgroup_fj_stress_net_prio_3_3_each cgroup_fj_stress.sh net_prio 3 3 each
+#cgroup_fj_stress_net_prio_4_4_each cgroup_fj_stress.sh net_prio 4 4 each
+#cgroup_fj_stress_net_prio_2_9_each cgroup_fj_stress.sh net_prio 2 9 each
+#cgroup_fj_stress_net_prio_10_3_each cgroup_fj_stress.sh net_prio 10 3 each
+#cgroup_fj_stress_net_prio_1_200_each cgroup_fj_stress.sh net_prio 1 200 each
+#cgroup_fj_stress_net_prio_200_1_each cgroup_fj_stress.sh net_prio 200 1 each
 
 # Stress test for hugetlb cgroup
-cgroup_fj_stress_hugetlb_2_2_none cgroup_fj_stress.sh hugetlb 2 2 none
-cgroup_fj_stress_hugetlb_3_3_none cgroup_fj_stress.sh hugetlb 3 3 none
-cgroup_fj_stress_hugetlb_4_4_none cgroup_fj_stress.sh hugetlb 4 4 none
-cgroup_fj_stress_hugetlb_2_9_none cgroup_fj_stress.sh hugetlb 2 9 none
-cgroup_fj_stress_hugetlb_10_3_none cgroup_fj_stress.sh hugetlb 10 3 none
-cgroup_fj_stress_hugetlb_1_200_none cgroup_fj_stress.sh hugetlb 1 200 none
-cgroup_fj_stress_hugetlb_200_1_none cgroup_fj_stress.sh hugetlb 200 1 none
-
-cgroup_fj_stress_hugetlb_2_2_one cgroup_fj_stress.sh hugetlb 2 2 one
-cgroup_fj_stress_hugetlb_3_3_one cgroup_fj_stress.sh hugetlb 3 3 one
-cgroup_fj_stress_hugetlb_4_4_one cgroup_fj_stress.sh hugetlb 4 4 one
-cgroup_fj_stress_hugetlb_2_9_one cgroup_fj_stress.sh hugetlb 2 9 one
-cgroup_fj_stress_hugetlb_10_3_one cgroup_fj_stress.sh hugetlb 10 3 one
-cgroup_fj_stress_hugetlb_1_200_one cgroup_fj_stress.sh hugetlb 1 200 one
-cgroup_fj_stress_hugetlb_200_1_one cgroup_fj_stress.sh hugetlb 200 1 one
-
-cgroup_fj_stress_hugetlb_2_2_each cgroup_fj_stress.sh hugetlb 2 2 each
-cgroup_fj_stress_hugetlb_3_3_each cgroup_fj_stress.sh hugetlb 3 3 each
-cgroup_fj_stress_hugetlb_4_4_each cgroup_fj_stress.sh hugetlb 4 4 each
-cgroup_fj_stress_hugetlb_2_9_each cgroup_fj_stress.sh hugetlb 2 9 each
-cgroup_fj_stress_hugetlb_10_3_each cgroup_fj_stress.sh hugetlb 10 3 each
-cgroup_fj_stress_hugetlb_1_200_each cgroup_fj_stress.sh hugetlb 1 200 each
-cgroup_fj_stress_hugetlb_200_1_each cgroup_fj_stress.sh hugetlb 200 1 each
-
-controllers	test_controllers.sh
+#cgroup_fj_stress_hugetlb_2_2_none cgroup_fj_stress.sh hugetlb 2 2 none
+#cgroup_fj_stress_hugetlb_3_3_none cgroup_fj_stress.sh hugetlb 3 3 none
+#cgroup_fj_stress_hugetlb_4_4_none cgroup_fj_stress.sh hugetlb 4 4 none
+#cgroup_fj_stress_hugetlb_2_9_none cgroup_fj_stress.sh hugetlb 2 9 none
+#cgroup_fj_stress_hugetlb_10_3_none cgroup_fj_stress.sh hugetlb 10 3 none
+#cgroup_fj_stress_hugetlb_1_200_none cgroup_fj_stress.sh hugetlb 1 200 none
+#cgroup_fj_stress_hugetlb_200_1_none cgroup_fj_stress.sh hugetlb 200 1 none
+
+#cgroup_fj_stress_hugetlb_2_2_one cgroup_fj_stress.sh hugetlb 2 2 one
+#cgroup_fj_stress_hugetlb_3_3_one cgroup_fj_stress.sh hugetlb 3 3 one
+#cgroup_fj_stress_hugetlb_4_4_one cgroup_fj_stress.sh hugetlb 4 4 one
+#cgroup_fj_stress_hugetlb_2_9_one cgroup_fj_stress.sh hugetlb 2 9 one
+#cgroup_fj_stress_hugetlb_10_3_one cgroup_fj_stress.sh hugetlb 10 3 one
+#cgroup_fj_stress_hugetlb_1_200_one cgroup_fj_stress.sh hugetlb 1 200 one
+#cgroup_fj_stress_hugetlb_200_1_one cgroup_fj_stress.sh hugetlb 200 1 one
+
+#cgroup_fj_stress_hugetlb_2_2_each cgroup_fj_stress.sh hugetlb 2 2 each
+#cgroup_fj_stress_hugetlb_3_3_each cgroup_fj_stress.sh hugetlb 3 3 each
+#cgroup_fj_stress_hugetlb_4_4_each cgroup_fj_stress.sh hugetlb 4 4 each
+#cgroup_fj_stress_hugetlb_2_9_each cgroup_fj_stress.sh hugetlb 2 9 each
+#cgroup_fj_stress_hugetlb_10_3_each cgroup_fj_stress.sh hugetlb 10 3 each
+#cgroup_fj_stress_hugetlb_1_200_each cgroup_fj_stress.sh hugetlb 1 200 each
+#cgroup_fj_stress_hugetlb_200_1_each cgroup_fj_stress.sh hugetlb 200 1 each
+
+#controllers	test_controllers.sh
 
 cpuacct_1_1 cpuacct.sh 1 1
 cpuacct_1_10 cpuacct.sh 1 10
diff --git a/runtest/mm b/runtest/mm
index f288fed36..01b2bcb62 100644
--- a/runtest/mm
+++ b/runtest/mm
@@ -92,12 +92,12 @@ vma03 vma03
 vma04 vma04
 vma05 vma05.sh
 
-overcommit_memory01 overcommit_memory
-overcommit_memory02 overcommit_memory -R 0
-overcommit_memory03 overcommit_memory -R 30
-overcommit_memory04 overcommit_memory -R 80
-overcommit_memory05 overcommit_memory -R 100
-overcommit_memory06 overcommit_memory -R 200
+#overcommit_memory01 overcommit_memory
+#overcommit_memory02 overcommit_memory -R 0
+#overcommit_memory03 overcommit_memory -R 30
+#overcommit_memory04 overcommit_memory -R 80
+#overcommit_memory05 overcommit_memory -R 100
+#overcommit_memory06 overcommit_memory -R 200
 
 max_map_count max_map_count -i 10
 
diff --git a/runtest/sched b/runtest/sched
index 172fe4174..9b8bc3884 100644
--- a/runtest/sched
+++ b/runtest/sched
@@ -7,12 +7,12 @@ time-schedule01		time-schedule
 trace_sched01		trace_sched -c 1
 
 cfs_bandwidth01 cfs_bandwidth01 -i 5
-hackbench01 hackbench 50 process 1000
-hackbench02 hackbench 20 thread 1000
-starvation starvation
+#hackbench01 hackbench 50 process 1000
+#hackbench02 hackbench 20 thread 1000
+#starvation starvation
 
 sched_cli_serv run_sched_cliserv.sh
 # Run this stress test for 2 minutes
-sched_stress sched_stress.sh
+#sched_stress sched_stress.sh
 
 autogroup01 autogroup01
diff --git a/runtest/syscalls b/runtest/syscalls
index 45e60341e..2ba7ff8ea 100644
--- a/runtest/syscalls
+++ b/runtest/syscalls
@@ -386,8 +386,9 @@ fork08 fork08
 fork09 fork09
 fork10 fork10
 fork11 fork11
-fork13 fork13
-fork14 fork14
+#Excessive runtime
+#fork13 fork13
+#fork14 fork14
 
 fpathconf01 fpathconf01
 
@@ -867,10 +868,11 @@ msgctl03 msgctl03
 msgctl04 msgctl04
 msgctl05 msgctl05
 msgctl06 msgctl06
-msgstress01 msgstress01
-msgstress02 msgstress02
-msgstress03 msgstress03
-msgstress04 msgstress04
+#These take aeons on qemu
+#msgstress01 msgstress01
+#msgstress02 msgstress02
+#msgstress03 msgstress03
+#msgstress04 msgstress04
 msgctl12 msgctl12
 
 msgget01 msgget01
diff --git a/runtest/syscalls-ipc b/runtest/syscalls-ipc
index df41140a7..3b80ac91e 100644
--- a/runtest/syscalls-ipc
+++ b/runtest/syscalls-ipc
@@ -4,10 +4,11 @@ msgctl03 msgctl03
 msgctl04 msgctl04
 msgctl05 msgctl05
 msgctl06 msgctl06
-msgstress01 msgstress01
-msgstress02 msgstress02
-msgstress03 msgstress03
-msgstress04 msgstress04
+#These take aeons on qemu
+#msgstress01 msgstress01
+#msgstress02 msgstress02
+#msgstress03 msgstress03
+#msgstress04 msgstress04
 msgctl12 msgctl12
 
 msgget01 msgget01
-- 
2.43.5

